app/controllers/runtime/syslog_drain_urls_controller.rb

```diff
@@ -10,20 +10,29 @@ class SyslogDrainUrlsController < RestController::BaseController
 
```

Comment:
The permission of the file is changed from `100644` to `100755`. Is it by design?